test/Microsoft.Azure.WebJobs.ServiceBus.UnitTests/Config/ServiceBusJobHostConfigurationExtensionsTests.cs

```diff
@@ -49,24 +53,28 @@ public void UseServiceBus_NoServiceBusConfiguration_PerformsExpectedRegistration
         [Fact]
         public void UseServiceBus_ServiceBusConfigurationProvided_PerformsExpectedRegistration()
         {
-            JobHostConfiguration config = new JobHostConfiguration();
+            string fakeConnStr = "test service bus connection";
 
-            IExtensionRegistry extensions = config.GetService<IExtensionRegistry>();
-            IExtensionConfigProvider[] configProviders = extensions.GetExtensions<IExtensionConfigProvider>().ToArray();
-            Assert.Empty(configProviders);
+            IHost host = new HostBuilder()
+                .ConfigureDefaultTestHost()
+                .AddServiceBus()
```

Comment:
I'd expect the pattern to be `.AddServiceBus(sbOptions)` similar to what we had before. Passing in the options to the AddServiceBus call. Doing this via a detached ConfigureServices call seems much less discoverable.

Reply:
We can add the config to the extension method. With the new pattern, you wouldn't pass the options, but provide a lambda to set configuration.